gcf/lib/path.go

```diff
@@ -151,3 +156,11 @@ func FindRootImportDirectory(triggerPath string) (string, error) {
 	}
 	return strings.Join(pathList[:len(pathList)-3], "/"), nil
 }
+
+// StripUntilRootDir trims one up folder of importRootDir from pathFromBucket.
+// Example:
+// StripUntilRootDir("some/path/to/root/data/source/dataset/a.tmcf", "some/path/to/root")
+// -> "root/data/source/dataset/a.tmcf"
+func StripUntilRootDir(pathFromBucket, importRootDir string) string {
```

Comment:
still not sure why need this. GCF computes "root" and find all children under "root", so the path info struct should contains {root, List[children_path]}.  I feel  processing can be simplified somewhere?

Reply:
Thanks for the pointer. This function is now removed.

CollectImportFiles now takes in paths, where each path is the entire gcs path without modifications, and datadir, which tells it where the source, datasets are.